### ISSUE TITLE
[🐛 Bug]: Missing dependency websocket-client in Python source package

### ISSUE THREAD

**Issue by rolim** [SeleniumHQ/selenium#14184]
### What happened?

When installing the source distribution of selenium-4.22.0, the required dependency "websocket-client>=1.8.0" is missing.

### How can we reproduce the issue?

```shell
$ pip3 install selenium --no-binary selenium
$ python3 -c "from selenium import webdriver"
```

### Relevant log output

```shell
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/tmp/venv/lib/python3.12/site-packages/selenium/webdriver/__init__.py", line 20, in <module>
    from .chrome.webdriver import WebDriver as Chrome  # noqa
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/tmp/venv/lib/python3.12/site-packages/selenium/webdriver/chrome/webdriver.py", line 18, in <module>
    from selenium.webdriver.chromium.webdriver import ChromiumDriver
  File "/tmp/venv/lib/python3.12/site-packages/selenium/webdriver/chromium/webdriver.py", line 22, in <module>
    from selenium.webdriver.remote.webdriver import WebDriver as RemoteWebDriver
  File "/tmp/venv/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py", line 67, in <module>
    from .websocket_connection import WebSocketConnection
  File "/tmp/venv/lib/python3.12/site-packages/selenium/webdriver/remote/websocket_connection.py", line 23, in <module>
    from websocket import WebSocketApp
ModuleNotFoundError: No module named 'websocket'
```

### Operating System

Ubuntu

### Selenium version

4.22.0

### What are the browser(s) and version(s) where you see this issue?

n/a

### What are the browser driver(s) and version(s) where you see this issue?

n/a

**Comment by titusfortner** [comment]
Ah, we need to add it to the  install requires section of the setup.py file

**Comment by Delta456** [comment]
I believe this issue can be closed because #14187 is merged.
